btax/run_btax.py

```diff
@@ -140,7 +140,6 @@ def run_btax_to_json_tables(test_run=False,start_year=2016,iit_reform=None,**use
         else:
             raise ValueError('Expected an "asset" or "industry" related table')
     print['base_output_by_asset']
-    quit()
```

Comment:
@talumbau Are you sure you pulled the most recent code?  The latest commits remove this `quit`.  You are getting SystemExit error from this quit.  Not sure where it came from.